### ISSUE TITLE
[4.8 beta] Add a standing line indicator that visualizes the Z-seam X and Y coordinate.

### ISSUE THREAD

**Issue by printingotb** [Ultimaker/Cura#8662]
**Is your feature request related to a problem? Please describe.**
I saw this: https://github.com/Ultimaker/Cura/pull/8599 and I hope it becomes reality soon.
Either way, I think it would be a good idea to also have a standing line/bar that indicates where on the build-plate the Z-seam coordinates are.

**Describe the solution you'd like**
Add a standing line at the Z-seam coordinate reaching from Z0 to Zmax. This will make it much easier to figure out what coordinates to enter to get the seam in the best position.
Preferably the user can select and drag the indicator around. But even if it's not drag-able, I believe this indicator will be helpful.

The indicator should have a check-box to turn visibility on and off in the colour scheme box.
If it can be drag-able, it can be another option in the Z-seam alignment dropdown, called "Indicator" that updates the coordinates when moved. If there are many models on the buildplate and the Z-seam is set to relative, there would be many indicators, not sure if that is a problem or if it should only be visible when a model is selected.

**Describe alternatives you've considered**
So far I have been enabling coasting to see the seam, then change the coordinates and re-slice until it's good. And then forgetting to turn off coasting before starting the print :sweat_smile:

**Affected users and/or printers**
Everyone.

**Additional context**
Here I imported a Ø0.4mm cylinder and set it to block supports just to get an example of visualization, coasting is on to show the seam. (Don't mind that the benchy is missing some skin areas):
![1](https://user-images.githubusercontent.com/32172100/97719577-852c0200-1ac7-11eb-97b9-098e74d76f79.PNG)
![2](https://user-images.githubusercontent.com/32172100/97719585-878e5c00-1ac7-11eb-9828-cd3426b3bad2.PNG)

**Comment by Ghostkeeper** [comment]
Duplicate of https://github.com/Ultimaker/Cura/issues/2636.
